### DISCUSSION TITLE
What is the difference between Free and Premium Edition ?

### DISCUSSION THREAD

**Discussion by IkelAtomig** [filecxx/FileCentipede#773] [General]
It looks kinda same to me.

**Comment by filecxx** [comment]
> It looks kinda same to me.

See (1.9 - 2.82):
http://www.filecxx.com/en_US/update_log.html

**Reply by IkelAtomig** [reply]
Really ?

It would be better if you distinguish them in the webpage. It is difficult and time consuming task to get some idea of it.

Would have been better if you maintained a list of first. Not blaming though.
